docs/API.md

```diff
@@ -364,6 +372,23 @@ Returns:
 * rejects with:
     * any error from the underlying storage system (wrapped in `error.wrap()`)
 
+## .verifyToken(tokenVerificationId, accountData)
+
+Verifies sessionTokens.
+Note that it takes the tokenVerificationId
+specified when creating the token,
+NOT the tokenId.
+`accountData` is an object
+with a `uid` property.
+
+Returns a promise that:
+
+* resolves with an object `{}`
+  (whether a row was updated or not,
```

Comment:
nit: when I first read this, I thought it somehow indicated whether or not a row was updated; saying "regardless of whether a row was updated or not" may be clearer